wire-schema/build.gradle.kts

```diff
@@ -143,3 +151,108 @@ if (project.rootProject.name == "wire") {
     }
   }
 }
+
+with (CoreLoaderTasks) {
+  installCoreLoaderTasks()
+}
+
+
+/*
+ * When we add Wire's `.proto` files like `google/descriptor.proto` as an element on the class path,
+ * we risk colliding with other `.jar` files that do the same, including other versions of Wire
+ * and with `protobuf-java.jar`.
+ *
+ * Mitigate this by putting Wire's copy of these protos in a directory named by the hash of its
+ * content.
+ *
+ * Also generate a single `.kt` file with this hash, so we can look it up later.
+ */
+fun Project.installCoreLoaderTasks() {
```

Comment:
I couldn’t move this to the build-support subproject because we need to build wire-schema before we build that!